test/framework/easyconfigs/yeb/Python-2.7.10-ictce-4.1.13.yeb

```diff
@@ -0,0 +1,132 @@
+_internal_variables_:
+    - &numpyversion 1.9.2
+    - &scipyversion 0.15.1
+    - &version 2.7.10
+
+easyblock: ConfigureMake
+
+name: Python
+version: *version
+
+homepage: http://python.org/
+description: |
+    Python is a programming language that lets you work more quickly and integrate your systems
+    more effectively.
+
+toolchain: {name: ictce, version: 4.1.13}
+toolchainopts: {pic: True, opt: True, optarch: True}
+
+source_urls: [!join ["http://www.python.org/ftp/python/", *version, /]]
```

Comment:
use `%(version)s` here, and drop the `version` anchor/alias, there's no reason that this should be different from the `.eb` easyconfig?